mne_qt_browser/_pg_figure.py

```diff
@@ -1697,10 +1697,10 @@ def __init__(self, main, **kwargs):
 class ProjDialog(_BaseDialog):
     """A dialog to toggle projections."""
 
-    def __init__(self, main, *, name):
+    def __init__(self, main, title='Projectors', **kwargs):
         self.external_change = True
         # Create projection-layout
-        super().__init__(main.window(), name=name, title='Projectors')
+        super().__init__(main.window(), title=title, **kwargs)
```

Comment:
See above, please revert this

Reply:
Attended 0408af1349194cae0016f598f2e85529846728b3